source/common/config/new_grpc_mux_impl.cc

```diff
@@ -0,0 +1,238 @@
+#include "common/config/new_grpc_mux_impl.h"
```

Comment:
this file doesn't have test coverage in many error cases: https://279265-65214191-gh.circle-artifacts.com/0/coverage/coverage/proc/self/cwd/source/common/config/new_grpc_mux_impl.cc.html

Reply:
Done. The ratelimit-related ones are handled in grpc_mux_impl_test, so the unification PR should take care of those.